src/lib/components/schedule/schedule-basic-frequency.svelte

```diff
@@ -22,6 +22,6 @@
 
 <style lang="postcss">
   .frequency {
-    @apply flex h-auto max-h-32 flex-col overflow-auto rounded-lg bg-inverse px-2 py-2 font-mono text-sm text-[#e9d5ff];
+    @apply flex h-auto max-h-32 flex-col overflow-auto rounded-lg bg-spaceBlack px-2 py-2 font-mono text-sm text-[#e9d5ff];
```

Comment:
Should we just make `spaceBlack` … `black`?

Reply:
I would but we also have a "true" [black](https://github.com/temporalio/ui/pull/2118/files#diff-2f57d51c15b2cb580beecdf8d11211275f39e4e4bfb05301e4f919d3b3a92408R150) that we use.